DOCKER.md

```diff
@@ -71,7 +71,7 @@ You can test if this will work for you by executing the following steps
 
 Start the accumulo-proxy container and enter it
 ```commandline
-docker run -it --rm -p 42424:42424 --network="host" --name accumulo-proxy accumulo-proxy:latest bash;
+docker run -it --rm -p 42424:42424 --network="host" --name accumulo-proxy accumulo-proxy:latest sh;
```

Comment:
I'm not sure the impact of this, but I do know a lot of Accumulo scripts use bash specifically.